src/Symfony/Component/VarExporter/Tests/LazyGhostTraitTest.php

```diff
@@ -438,6 +439,26 @@ public function testReadOnlyClass()
         $this->assertSame(123, $proxy->foo);
     }
 
+    public function testAccessingUninializedPropertyWithoutLazyGhost()
+    {
+        $this->expectException(\Error::class);
+        $this->expectExceptionCode(0);
+        $this->expectExceptionMessage('Typed property Symfony\Component\VarExporter\Tests\Fixtures\LazyGhost\ClassWithUninitializedObjectProperty::$property must not be accessed before initialization');
+
+        $object = new ClassWithUninitializedObjectProperty();
```

Comment:
let's move this line before the expect calls (same below)

Reply:
it is done